src/shortcuts/toggleSort.tsx

```diff
@@ -42,8 +43,7 @@ const toggleSortShortcut: Shortcut = {
     const sortPreference = globalSort === 'Alphabetical' ? 'None' : 'Alphabetical'
 
     if (cursor) {
```

Comment:
Find a way to do this with a single `dispatch` call (i.e. with a ternary statement inside the dispatch argument). The logic will be clearer if it's lower down since the two paths have so much in common.